Packs/Legacy/Integrations/integration-ArcherRSA.yml

```diff
@@ -1467,7 +1467,13 @@ script:
 
             var fields = verifyOrMakeArray(record.Field);
             fields.forEach(function (field) {
-                toContext.Fields[GLOBAL_MAPPING[field['-id']].Name] = field['#text'];
+                if (field['#text'] != null) {
+                    toContext.Fields[GLOBAL_MAPPING[field['-id']].Name] = field['#text'];
+                } else if (field.ListValues != null && field.ListValues['#text'] != null) {
+                    toContext.Fields[GLOBAL_MAPPING[field['-id']].Name] = field.ListValues['#text'];
+                } else {
+                    toContext.Fields[GLOBAL_MAPPING[field['-id']].Name] = field['#text'];
```

Comment:
this seems identical to the first condition

Reply:
Fixed now - did not change the logic (this condition is  equivalent to null at this point, which is the desired behavior) but wrote `null` explicitly for code readability and clarity